packages/opencensus-web-instrumentation-zone/src/interaction-tracker.ts

```diff
@@ -153,20 +148,24 @@ export class InteractionTracker {
       },
       kind: SpanKind.UNSPECIFIED,
     };
-    // Start a new RootSpan for a new user interaction.
-    tracing.tracer.startRootSpan(spanOptions, root => {
-      // As startRootSpan creates the zone and Zone.current corresponds to the
-      // new zone, we have to set the currentEventTracingZone with the Zone.current
-      // to capture the new zone.
-      this.currentEventTracingZone = Zone.current;
+    // Start a new RootSpan for a new user interaction, also, creates the new zone
+    // from the coming task.zone.
```

Comment:
What would you think about re-working `from the coming task.zone` to `forked from the original task.zone`?